docs/user-guide/model-inputs.md

```diff
@@ -224,20 +224,22 @@ Thus, command-line arguments override settings in the configuration file, and co
 
 ### Model Flags
 
-| Option        | Default | Description                                                                              |
-| ------------- | ------- | ---------------------------------------------------------------------------------------- |
-| `events`      | on      | Enable event handling.                                                                   |
-| `gdd`         | on      | Use growing degree days to determine leaf growth.                                        |
-| `growth-resp` | off     | Explicitly model growth respiration, rather than including with maintenance respiration. |
-| `leaf-water`  | off     | Calculate leaf pool and evaporate from that pool.                                        |
-| `litter-pool` | off     | Enable litter pool in addition to single soil carbon pool.                               |
-| `microbes`    | off     | Enable microbe modeling.                                                                 |
-| `snow`        | on      | Keep track of snowpack, rather than assuming all precipitation is liquid.                |
-| `soil-phenol` | off     | Use soil temperature to determine leaf growth.                                           |
-| `water-hresp` | on      | Whether soil moisture affects heterotrophic respiration.                                 |
+| Option           | Default | Description                                                                             |
+|------------------|---------|-----------------------------------------------------------------------------------------|
+| `events`         | on      | Enable event handling                                                                   |
+| `gdd`            | on      | Use growing degree days to determine leaf growth                                        |
+| `growth-resp`    | off     | Explicitly model growth respiration, rather than including with maintenance respiration |
+| `leaf-water`     | off     | Calculate leaf pool and evaporate from that pool                                        |
+| `litter-pool`    | off     | Enable litter pool in addition to single soil carbon pool                               |
+| `microbes`       | off     | Enable microbe modeling                                                                 |
+| `nitrogen-cycle` | off     | Enable nitrogen cycle modeling                                                          |
+| `snow`           | on      | Keep track of snowpack, rather than assuming all precipitation is liquid                |
+| `soil-phenol`    | off     | Use soil temperature to determine leaf growth                                           |
+| `water-hresp`    | on      | Whether soil moisture affects heterotrophic respiration                                 |
 
 Note the following restrictions on these options:
  - `soil-phenol` and `gdd` may not both be turned on
+ - `events` and `microbes` may not both be turned on
```

Comment:
Should we also prohibit nitrogen + microbes both on? I suspect it would be complicated and at this point unnecessary to support the combination.